super_team_picker/db/migrations/20191215184803_createCohorts.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function(knex) {
+    return knex.schema.createTable("cohorts", table => {
+        table.increments('id');
+        table.string('name');
+        table.text('members');
+        table.string('logoUrl');
```

Comment:
Use `table.text` instead of string to use a data type for the `logoUrl` that accepts a length that's greater than 255, since image urls can get long.